lib/utils/utils.js

```diff
@@ -270,3 +275,62 @@ function findUnorderedProperty(arr) {
 
   return null;
 }
+
+/**
+ * Checks whether a computed property has duplicate dependent keys.
+ *
+ * @param  {CallExpression} callExp Given call expression
+ * @return {Boolean}        Flag whether dependent keys present.
+ */
+function hasDuplicateDependentKeys(callExp) {
```

Comment:
This function should be in `ember.js` util. We want these utils to be independent from ember-related stuff.